sdk/core/core-amqp/test/requestResponse.spec.ts

```diff
@@ -234,19 +158,22 @@ describe("RequestResponseLink", function() {
           body: "Hello World!!"
         }
       });
-    }, 4000);
+    }, 2000);
     try {
       const controller = new AbortController();
-      const signal = controller.signal;
-      setTimeout(controller.abort.bind(controller), 100);
-      await link.sendRequest(request, {
-        delayInSeconds: 1,
-        timeoutInSeconds: 5,
-        abortSignal: signal // cancel between request attempts
-      });
+      const signal: AbortSignalLike = controller.signal;
+      const onAborted = () => {
+        // do nothing and just to mark signal as aborted internally
+      };
+      signal.addEventListener("abort", onAborted);
```

Comment:
We don't need to add the event listener here. Calling `controller.abort()` should be enough to fire the signal

Reply:
Oh, I thought only if a listener was added, the internal map gets updated.
Will try this out and see